components/frontend/src/app/projects/[name]/sessions/[sessionName]/components/explorer/explorer-panel.tsx

```
@@ -0,0 +1,144 @@
+      {/* Tab header */}
+      <div className="flex items-center justify-between border-b px-1">
+        <div className="flex">
+          <button
+            onClick={() => onTabChange("files")}
+            className={cn(
+              "px-3 py-2 text-sm font-medium transition-colors flex items-center gap-1.5",
+              activeTab === "files"
+                ? "text-foreground border-b-2 border-primary"
+                : "text-muted-foreground hover:text-foreground",
+            )}
+          >
```

Biome (2.4.7)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)